src/components/app/app.ts

```diff
@@ -0,0 +1,25 @@
+import AppController from '../controller/controller';
+import { AppView } from '../view/appView';
+
+class App {
+    private controller: AppController;
+    private view: AppView;
+
+    constructor() {
+        this.controller = new AppController();
+        this.view = new AppView();
+    }
+
+    start(): void {
+        const sourcesElement = document.querySelector('.sources') as HTMLElement | null;
```

Comment:
Considering that in line 16 you make a check for` sourcesElement`  being null, casting ` sourcesElement`  as HTMLElement | null is redundant